src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -3077,42 +3065,45 @@ static WellKnownMembers()
                      (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Int32,
                      (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Boolean,
 
-                 // System_Index__IsFromEnd
-                 (byte)(MemberFlags.Property),                                                                                                                  // Flags
+                 // System_Index__GetOffset
```

Comment:
>System_Index__GetOffset [](start = 20, length = 23)

I expected some tests to do `MakeTypeMissing(System_Index__GetOffset)` but couldn't find any. Do we have coverage for missing types?